Microsoft.Toolkit.Uwp.UI.Controls/FlexPanel/FlexPanel.cs

```diff
@@ -0,0 +1,1613 @@
+        {
+            get => (FlexDirection)GetValue(DirectionProperty);
+            set => this.SetNewValue(DirectionProperty, value);
+        }
+
+        /// <summary>
+        /// Dependency Property for the FlexPanel.JustifyContent property
+        /// </summary>
+        public static readonly DependencyProperty JustifyContentProperty = DependencyProperty.Register(
+            nameof(JustifyContent),
+            typeof(FlexJustify),
+            typeof(FlexPanel),
+            new PropertyMetadata(FlexItem.JustifyContentDefault, new PropertyChangedCallback((d, e) => ((FlexPanel)d).OnJustifyContentChanged(e))));
+
+        private void OnJustifyContentChanged(DependencyPropertyChangedEventArgs e)
+        {
+            if (_root != null)
+            {
+                _root.JustifyContent = (FlexJustify)e.NewValue;
+                InvalidateMeasure();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets how the FlexPanel will distribute space between and around child items
+        /// along the main-axis.
+        /// </summary>
+        /// <remarks>The default value for this property is Start.</remarks>
+        public FlexJustify JustifyContent
+        {
+            get => (FlexJustify)GetValue(JustifyContentProperty);
+            set => this.SetNewValue(JustifyContentProperty, value);
+        }
+
+        /// <summary>
+        /// The Dependency Property for the FlexPanel.Wrap property
+        /// </summary>
+        public static readonly DependencyProperty WrapProperty = DependencyProperty.Register(
+            nameof(Wrap),
+            typeof(FlexWrap),
+            typeof(FlexPanel),
+            new PropertyMetadata(FlexItem.WrapDefault, new PropertyChangedCallback((d, e) => ((FlexPanel)d).OnWrapChanged(e))));
+
+        private void OnWrapChanged(DependencyPropertyChangedEventArgs e)
+        {
+            if (_root != null)
+            {
+                _root.Wrap = (FlexWrap)e.NewValue;
+                InvalidateMeasure();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets whether child elements should be laid out in a single line(NoWrap)
+        /// or multiple lines(Wrap or WrapReverse). If this property is set to Wrap or WrapReverse,
+        /// <see cref = "P:Microsoft.Toolkit.Uwp.UI.Controls.FlexItem.AlignContent" /> can then be
+        /// used to specify how the lines should be distributed.
+        /// </summary>
+        /// <remarks>The default value for this property is NoWrap.</remarks>
+        public FlexWrap Wrap
+        {
+            get => (FlexWrap)GetValue(WrapProperty);
+            set => this.SetNewValue(WrapProperty, value);
+        }
+
+        /// <summary>
+        /// The Attached Dependency Property for the FlexLayout.Order attached property
+        /// </summary>
+        public static readonly DependencyProperty OrderProperty = DependencyProperty.RegisterAttached(
+            "Order",
```

Comment:
Use `nameof` for all DependencyProperty names instead of strings.
`nameof(Order)`

Reply:
Good catch!

Reply:
Correction:  There is not an Order property - Order is an attached property - so there is not a variable to which nameof can be applied.

Reply:
We should split out all the properties into a separate .Properties.cs partial class like we do for our other controls.